flow/FileTraceLogWriter.h

```diff
@@ -26,6 +26,7 @@
 #include "flow/FastRef.h"
 #include "flow/Trace.h"
 
+#include <set>
```

Comment:
Is this include required anymore?

Reply:
Probably not anymore, after the same include added in Trace.h

Reply:
Done.